pkg/controller/storagecluster/noobaa_system_reconciler_test.go

```diff
@@ -395,3 +395,104 @@ func getReconciler(t *testing.T, objs ...runtime.Object) ReconcileStorageCluster
 		platform: &Platform{},
 	}
 }
+
+type testKMSArgs struct {
+	kmsProvider string
+	kmsAddress  string
+	testLabel   string
+	valid       bool
```

Comment:
Since this is almost always true, I'd suggest changing it to something like `expectedFailure` and only setting it to true in the invalid cases.

Reply:
Done